scripts/github_runner/setup_bare_metal_runner.sh

```diff
@@ -37,6 +37,7 @@ GH_PAT=$2
 SANDBOX=$3
 
 VHIVE_ROOT="$(git rev-parse --show-toplevel)"
+rm /etc/systemd/system/actions.runner.vhive-serverless-vhive.*.service
```

Comment:
```suggestion
rm /etc/systemd/system/actions.runner.vhive-serverless-vhive.*.service || echo No need to clean the node
```

provision for the clean node otherwise the script would fail